### PR TITLE
SOC-1444 Don't allow posting new messages via GET

### PR DESCRIPTION
**Pull request by garthwebb**
Require POST and an edit token

https://wikia-inc.atlassian.net/browse/SOC-1444

### CODE REVIEW

**Comment by armonr** [comment]
:+1:

File: extensions/wikia/Wall/js/WallBackendBridge.js [hunk 1]
```diff
@@ -50,7 +50,8 @@
 					pagetitle: page.title,
 					pagenamespace: page.namespace,
 					convertToFormat: convertToFormat,
-					relatedTopics: relatedTopics
+					relatedTopics: relatedTopics,
+					token: window.mw.user.tokens.get('editToken')
 				},
 				callback: this.proxy(function (data) {
```
Comment (kvas-damian) [inline extensions/wikia/Wall/js/WallBackendBridge.js:56]: Does this callback support error message returned when token is not valid?

Comment (garthwebb) [inline extensions/wikia/Wall/js/WallBackendBridge.js:56]: See my note on this in #8854

**Comment by kvas-damian** [comment]
It's not optimal from UI perspective, but we're going to replace Forum and Wall by new Discussion product.

LGTM

**Pull request merged**